src/scripts/mutual_checker.css

```
@@ -6,3 +6,7 @@ svg.xkit-mutual-icon {
   margin-left: 0;
 	margin-right: 0.5ch;
 }
+
+.xkit-mutual-checker-hidden article {
```

If the option is only supposed to hide non-mutual posts on the dashboard, we need to make this selector more specific. Tumblr has `data-timeline` attributes on each timeline, which makes this very easy!

```suggestion
[data-timeline="/v2/timeline/dashboard"] .xkit-mutual-checker-hidden article {
```